mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule LatticeObserver.MixProject do
   def project do
     [
       app: :lattice_observer,
-      version: "0.1.0",
+      version: "0.2.0",
```

Comment:
Could this be a patch bump, or is it a breaking change?

Reply:
It actually is a breaking change, I tried to update `wadm` independently of the lattice-observer and `mix` required that they be on the same semver compatible version